testcases/master/template-android.jinja2

```diff
@@ -0,0 +1,17 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% set OS_INFO = OS_INFO|default("android") %}
+{% set DEPLOY_OS = DEPLOY_OS|default("android") %}
```

Comment:
Maybe these two variables should be defined somewhere else and not in the test?

Reply:
I am not sure if there is any other better places, but this file is the root file that all android jobs should extend, so I defined them here.